lib/cyclonedx/report_node_modules_cyclonedx.rb

```diff
@@ -6,5 +6,12 @@ class ReportNodeModules < Base
     def initialize(scan_report)
       super(scan_report)
     end
+
+    def build_component(dependency)
```

Comment:
Can you please add the comment above the method definition instead of writing in the body? I think it would be easier for other developers to read the code.

```
# Add scope if available:
# https://cyclonedx.org/docs/1.3/json/#components_items_scope
def build_component(dependency)
   ....
end
```